NGS2_WITNESS_Cycle1_confirmatory_exp1.R

```diff
@@ -0,0 +1,73 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. in March 2017
+
+# Load data for analysis
+exp1_cooperation <- read.csv('NGS2-Cycle1-Experiment1/cooperation_exp1.csv',
+                             header = TRUE, sep = ',')
+exp1_rewire <- read.csv('NGS2-Cycle1-Experiment1/rewire_exp1.csv', header = TRUE,
+                        sep = ',')
+
+### Show stopper
+
+print("ATENTION!! Need to make sure the exp1_cooperation dataset includes the following variables: sessionnum, condition, decision..0.D.1.C.")

```

Comment:
Why not program this as a condition so the script stops if the condition isn't met? Something like:
```
if(!all(c('sessionnum', 'condition', 'decision..0.D.1.C.') %in% names(exp1_cooperation))) {
    stop('Attention! `exp1_cooperation` must include `sessionnum`, `condition`, and `decision..0.D.1.C.` to proceed. Please fix.')
}
```

That way, if the variables aren't present, the script -- when `source()`'d will break and not continue until fixed up. Seems like that might be better, more specific behavior.

Reply:
Awesome, done!